src/main/java/frc/robot/Robot.java

```diff
@@ -87,7 +90,7 @@ public void autonomousInit() {
    */
   @Override
   public void autonomousPeriodic() {
-    Scheduler.getInstance().run();
+    // Scheduler.getInstance().run();
```

Comment:
I think we need this uncommented for when we start using commands.
```suggestion
    Scheduler.getInstance().run();
```